.buildkite/pipeline-release-helm.yml

```diff
@@ -1,5 +1,6 @@
 env:
   HELM_DRY_RUN: ${HELM_DRY_RUN:-true}
+  HELM_FORCE: ${HELM_FORCE:-false}
```

Comment:
Maybe also update `trigger-bk-release.sh` to pass the flag?

Reply:
Yes, without the typo https://github.com/elastic/cloud-on-k8s/pull/8771/files#r2239926733 :)